src/test/java/com/google/cloud/tools/gradle/appengine/core/DeployTargetResolverTest.java

```diff
@@ -5,26 +5,23 @@
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
  *
- *         http://www.apache.org/licenses/LICENSE-2.0
+ *          http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Were these two changes (here and L15) intentional?

Reply:
No, I don't know what happened, it might be the license file I have in intellij? I'll revert them for now.